netx/eventsaving/eventsaving.go

```diff
@@ -0,0 +1,78 @@
+// Package eventsaving uses measurable to save events.
+package eventsaving
+
+import (
+	"context"
+	"crypto/tls"
+	"net"
+	"net/http"
+	"sync"
+	"time"
+
+	"github.com/ooni/probe-engine/netx/measurable"
+)
+
+// An Event describes something that occurred.
+type Event struct {
+	Op   string
+	Time time.Time
+}
```

Comment:
It is probably more convenient to be able to wrap every Dialer independently of, say, a Resolver, because this gives us more flexibility. Also, a best practice of Go is to keep interfaces small, so...